pykern/pkio.py

```diff
@@ -128,6 +129,21 @@ def has_file_extension(filename, to_check):
     return e in to_check
 
 
+def is_pure_text(filename, test_size=512):
+    """Uses pykern.util.is_pure_text to guess whether file is pure text
+
+    Args:
+        filename (str|py.path.local): file to check
+        test_size (int): size of bytes chunk being tested
```

Comment:
language not right

Reply:
is "size of the bytes chunk read from filename" better?

Reply:
Or "size of the bytes chunk read from the start of filename"?